edx_sysadmin/apps.py

```diff
@@ -17,11 +17,18 @@ class EdxSysAdminConfig(AppConfig):
             "lms.djangoapp": {
                 "namespace": "sysadmin",
                 "regex": "^sysadmin/",
-            }
+            },
+            "cms.djangoapp": {
+                "namespace": "sysadmin",
+                "regex": "^sysadmin/",
```

Comment:
I was assuming that we won't need to add this URL routing for sysadmin in CMS since we have been accessing that through LMS only. Does only adding https://github.com/mitodl/edx-sysadmin/pull/72/files#diff-60f61ab7a8d1910d86d9fda2261620314edcae5894d5aaa236b821c7256badd7R93-R95 not work for accessing CMS functionality?

Reply:
Yes, It is working. Making the change.